src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import ball.characteristics.Ball;
+import ball.characteristics.Color;
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Creating a new instance of `Random` every time the method is called can be inefficient. Consider making it a private static final field of the class.

Reply:
not fixed

Reply:
same for this Random, it should be a field